quadrat/sass/templates/_header.scss

```diff
@@ -16,7 +20,7 @@
 
 		@include break-small-only(){
 			flex-basis: 100%;
-			margin: 20px 0;
+			margin: 70px 0;
```

Comment:
```suggestion
			margin: 45px 0;
```

I miscalculated this value — it should be 45px instead of 70px. But once we update that (and make the same change in Geologist, I think this is good to go.